Thermostat/download-and-run.sh

```diff
@@ -0,0 +1,16 @@
+echo "Updating System Resources"
+sudo yum -y update
+
+echo "Installing Git"
+sudo yum -y install git
```

Comment:
I’m Rick Harrison and this is my pawn shop. I work here with my old man and my son, Big Hoss, and in 23 years I’ve learned one thing. You never know what is gonna come through that door.